scripts/ps.py

```diff
@@ -64,6 +75,7 @@ def main():
                 int(pinfo['memory_info'].rss / 1024) or '?'
             memp = pinfo['memory_percent'] and \
                 round(pinfo['memory_percent'], 1) or '?'
+            stat = PROC_STATUSES_RAW.get(pinfo['status'], '')
```

Comment:
please rename the variable to "status". "stat" is a stdlib module and it's bad practice to obfuscate "reserved" names.